packages/cli/package.json

```diff
@@ -40,16 +40,15 @@
   },
   "dependencies": {
     "@basemaps/config": "^7.0.0",
+    "@basemaps/config-loader": "^7.0.0",
     "@basemaps/geo": "^7.0.0",
     "@basemaps/server": "^7.0.0",
     "@basemaps/shared": "^7.0.0",
     "@basemaps/tiler": "^7.0.0",
     "@basemaps/tiler-sharp": "^7.0.0",
-    "@chunkd/fs": "^10.0.2",
-    "@chunkd/source-aws-v2": "^9.3.1",
-    "@cogeotiff/core": "^7.2.0",
-    "@cotar/core": "^5.4.0",
-    "@cotar/tar": "^5.4.1",
+    "@chunkd/source-memory": "^11.0.2",
```

Comment:
I think all `@chunkd` library are from `shared` package now?

Reply:
the CLI needs more packages than what shared provides.